airflow/www/templates/airflow/dags.html

```diff
@@ -233,11 +233,32 @@ <h2>DAGs</h2>
       const DAGS_INDEX = "{{ url_for('admin.index') }}";
       const ENTER_KEY_CODE = 13;
 
+      function createOrAppendToQueryString(parameter, value) {
+        var query_string_parameters = {};
+        if (typeof window.location.search !== "undefined") {
+          window.location.search.substr(1).split('&').forEach(function (query_string_parameter) {
+            query_string_parameter_delimeted = query_string_parameter.split('=');
+            query_string_key = query_string_parameter_delimeted.shift();
+            query_string_value = query_string_parameter_delimeted.join('=');
+            query_string_parameters[query_string_key] = query_string_value
+          });
+        }
+        if (window.location.search == "") {
+          query_string_parameters = {};
```

Comment:
I can not imagine the case when this line does something. This variable already contains an empty object.